src/modules/critics/centralV.py

```diff
@@ -18,14 +18,33 @@ def __init__(self, scheme, args):
 
         # Set up network layers
         self.fc1 = nn.Linear(input_shape, args.hidden_dim)
-        self.fc2 = nn.Linear(args.hidden_dim, args.hidden_dim)
```

Comment:
Same comments I had for non centralised critics apply to centralised critic files